deps/npm/LICENSE

```diff
@@ -1,11 +1,29 @@
+The npm application
 Copyright (c) npm, Inc. and Contributors
-All rights reserved.
+Licensed on the terms of The Artistic License 2.0
 
-npm is released under the Artistic License 2.0, subject to additional terms
-that are listed below.
```

Comment:
Well that's where the attorneys get involved for dispute resolution... reading comprehension isn't limited to attorneys and/or arbitration but common sense says that the enforcement of the Artistic License says it can't be modified.